src/test/resources/features/users/update_users.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Update Users  
+
+  In order to update an account
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+
+  @functional
+  @createUser
+  Scenario: Verify that is possible to update the user information
+    When I send a POST request to "/user/edit/{id}" with the following form data
+      | username   | laura1  |
+      | name       | laura1  |
+      | lastName   | montaño |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "users/updateUser.json" JSON schema
+    And I validate that the response contains the following values
+      | status  | 200                                                    |
+      | message | "User edited" |
+
+  @negative
+  @createUser
+  Scenario:  Verify that is not possible to update the user information using an invalid id
+    When I send a POST request to "/user/edit/{id}" with following form data
+      | username   | laura1  |
+      | name       | laura1  |
+      | lastName   | montaño |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response contains the following values
+      | status  | 400                                                    |
+      | message | "The user with Id: {id}does not exist" |
```

Comment:
Please consider adding an empty line at the end of the file.

Reply:
Thanks, It's done.